pkg/types/ccipocr3/common_types.go

```diff
@@ -85,3 +86,29 @@ func (b *BigInt) UnmarshalJSON(p []byte) error {
 func (b BigInt) IsEmpty() bool {
 	return b.Int == nil
 }
+
+type P2PID [32]byte
+
+func (b P2PID) MarshalJSON() ([]byte, error) {
+	return json.Marshal(hex.EncodeToString(b[:]))
+}
+
+func (b *P2PID) UnmarshalJSON(data []byte) error {
+	var s string
+	if err := json.Unmarshal(data, &s); err != nil {
+		return err
+	}
+	decoded, err := hex.DecodeString(s)
+	if err != nil {
+		return err
+	}
+	copy(b[:], decoded)
+	return nil
+}
+
+func GetP2pID(id int) P2PID {
```

Comment:
What's this function for?

Reply:
Might not be a good place. I'm using it in tests. Will remove and add to tesethelpers in ccip.